### DISCUSSION TITLE
Looking for an example like this please.

### DISCUSSION THREAD

**Discussion by saisaigraph** [zauberzeug/nicegui#1793] [Ideas / Feature Requests]
Hi
Could you please provide an example like this using nicegui, below is the link:

https://blog.streamlit.io/auto-generate-a-dataframe-filtering-ui-in-streamlit-with-filter_dataframe/

Thanks
Sai

**Comment by falkoschindler** [comment]
@saisaigraph Can you, please, describe in more detail what you're looking for? The page you're referring to is pretty long and it's not obvious what your actual feature request is. Also, please give the discussion a meaningful title, so that others can find relevant information more easily. Thanks!

**Reply by saisaigraph** [reply]
Sorry about the brief description. 
If we have a dataframe which is completely denormalized and has data like Customers, Order, Items etc
the link i have provided gives filtering capabilities and automatically gives the filters with a single method call which is very useful for us. Please try to have a look at the screen shots provided in the link to get an idea. Please have a look at their demo:

https://st-filter-dataframe.streamlit.app/?ref=blog.streamlit.io
Please let me know if you need more info.
Thanks
Sai
